network-test-app/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM icr.io/codeengine/node:20-alpine
```

Comment:
I suggest to move to Node.js 22 right-away. Furthermore, adding a package upgrade helps to keep the vulnerabilities to a minimum
```suggestion
FROM icr.io/codeengine/node:22-alpine
RUN apk -U upgrade
```